open-api/rest-catalog-open-api.yaml

```diff
@@ -1610,13 +1610,27 @@ components:
 
     PageToken:
       description:
-        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables). 
-        Clients will initiate the first paginated request by sending an empty `pageToken` e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
-        signaling to the service that the response should be paginated.
+        An opaque token that allows clients to make use of pagination for list APIs
+        (e.g. ListTables). Clients may initiate the first paginated request by sending an empty
+        query parameter `pageToken` to the server e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
+        
+        Servers that support pagination should identify the `pageToken` parameter and return a
+        `next-page-token` in the response if there are more results available.  After the initial
+        request, the value of `next-page-token` from each response must be used as the `pageToken`
+        parameter value for the next request. The server must omit or return `null` value for the
+        `next-page-token` in the last response.
+        
+        Servers that support pagination must return all results in a single response with the value
+        of `next-page-token` omitted or set to `null` if the query parameter `pageToken` is not set in the
+        request.
+        
+        Servers that do not support pagination should ignore the `pageToken` parameter and return
+        all results in a single response. The `next-page-token` must be omitted or set to `null`
```

Comment:
Must be omitted.

Reply:
So above I think you mentioned to use `the service should send null.` but here you mentioned to use `must be omitted`. Just want to confirm which option we should say across the description to be consistent( I think omission makes sense to me)

Reply:
This is for services that do not support pagination, like all existing ones. In that case, they should not send the key. The difference is between not supporting pagination and ending paginated request exchanges.

Reply:
will fix this thanks for clarifying this.